app/pages/organisation/contextCreator.js

```diff
@@ -11,7 +11,7 @@ export const getContext = ({ organisation }) => ({
     if (organisation.address[key]) acc.push(organisation.address[key]);
     return acc;
   }, []) : [],
-  agreementSigned: organisation.isCatalogueAgreementSigned,
+  agreementSigned: organisation.catalogueAgreementSigned,
```

Comment:
This is for line 10-13. We are doing the address magic again. Perhaps a common function if we are doing it in more than one place